frontend/view-edit-client/client-home/view-edit-contact-info.component.tsx

```diff
@@ -106,7 +106,9 @@ export default function ViewEditContactInfo(props: ViewEditContactInfoProps) {
               <tr>
                 <td>Email:</td>
                 <td>
-                  <div>{client.email}</div>
+                  <div>
+                    {client.email === null ? "Not Provided" : client.email}
```

Comment:
Probably better to do a truthiness check here, in case the email is `undefined` or an empty string instead of null.

Good catch, though, finding this.

```suggestion
                    {client.email ? "Not Provided" : client.email}
```